testkit/src/main/scala/scorex/testkit/utils/FileUtils.scala

```diff
@@ -35,21 +33,22 @@ trait FileUtils {
     createTempDirForPrefix(prefix)
   }
 
+
+  @SuppressWarnings(Array("org.wartremover.warts.Recursion"))
```

Comment:
Please add ScalaDoc, in particular describing the rationale behind not deleting some of the files.

Reply:
Removed preserving hidden files (it was copy-pasted from the internets likely), also, made the deleteRecursive method internal to remove()